executor/show.go

```diff
@@ -527,10 +527,6 @@ func (e *ShowExec) fetchShowCreateTable() error {
 		buf.WriteString(fmt.Sprintf("  PRIMARY KEY (`%s`)", pkCol.Name.O))
 	}
 
-	if len(tb.Indices()) > 0 || len(tb.Meta().ForeignKeys) > 0 {
```

Comment:
if len(tb.Indices()) > 0 , we need a `\n`?

Reply:
Done